com.unity.cinemachine/Runtime/Behaviours/CinemachinePath.cs

```diff
@@ -98,10 +98,17 @@ float GetBoundingIndices(float pos, out int indexA, out int indexB)
             return pos;
         }
 
+#if CINEMACHINE_USE_BATCH_POINT_LINE_APIS
```

Comment:
It's not necessary to have API protectors in the runtime module, this change does not depend on new APIs